javascript/FieldToggleDescription.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Enable toggling (show/hide) of the field's description.
+ */
+
+(function ($) {
+
+    $.entwine('ss', function ($) {
+
+        $('.toggle-description').entwine({
```

Comment:
Should also be referenced (if not documented) here if this gets rolled in: http://doc.silverstripe.org/en/developer_guides/customising_the_admin_interface/how_tos/cms_formfield_help_text